api/container/containerv2/ingress_test.go

```diff
@@ -28,7 +28,7 @@ var _ = Describe("Ingress Secrets", func() {
 				server.AppendHandlers(
 					ghttp.CombineHandlers(
 						ghttp.VerifyRequest(http.MethodPost, "/ingress/v2/secret/createSecret"),
-						ghttp.VerifyJSON(`{"cluster":"bugi52rf0rtfgadjfso0","name":"testabc2","namespace":"","crn":"crn:v1:bluemix:public:cloudcerts:us-south:a/883079c85357a1f3f85d968780e56518:b65b5b7f-e904-4d2b-bd87-f0ccd57e76ba:certificate:333d8673f4d03c148ff81192b9edaafc","persistence":true}`),
+						ghttp.VerifyJSON(`{"cluster":"bugi52rf0rtfgadjfso0","name":"testabc2","namespace":"","crn":"crn:v1:bluemix:public:cloudcerts:us-south:a/883079c85357a1f3f85d968780e56518:b65b5b7f-e904-4d2b-bd87-f0ccd57e76ba:certificate:333d8673f4d03c148ff81192b9edaafc","persistence":true,"type":"","add":null}`),
```

Comment:
Do we need to send empty for type and add can we add omitemity for the fields

Reply:
>Do we need to send empty for type

I'm not sure I understand this question. The test case is not sending anything for the type parameter on the request. So this is the accurate representation of the request json. I can add the omitempty tags to the request but it will also not change this validation behavior, and has no benefit to us on the server side.